stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -470,11 +478,11 @@ function getAccessTokenFromAuthorizationRequest(ClientCredentialsGrantConfig|Pas
     return doRequest(tokenUrl, authorizationRequest, clientConfig, oauth2CacheEntry, clockSkewInSeconds);
 }
 
-# Request an access token from the authorization endpoint using the provided refresh configurations.
+# Requests an access token from the authorization endpoint using the provided refresh configurations.
 #
 # + config - Password grant type configuration or direct token configuration
 # + oauth2CacheEntry - OAuth2 cache entry
-# + return - Access token received or `Error` if an error occurred during HTTP client invocation
+# + return - Received access token or `oauth2:Error` if an error occurred during HTTP client invocation
```

Comment:
```suggestion
# + return - Received access token or `oauth2:Error` if an error occurred during HTTP client invocation
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - Received access token or else `oauth2:Error` if an error occurred while HTTP client invocation
```